ui/app/components/oidc/assignment-form.hbs

```diff
@@ -24,7 +24,6 @@
       @id="entities"
       @label="Entities"
       @placeholder="Search"
-      @renderInPlace={{true}}
```

Comment:
HDS components always have this as true, so following their guidelines I did so as well.